metadata-io/src/test/java/com/linkedin/metadata/search/elasticsearch/query/request/AggregationQueryBuilderTest.java

```diff
@@ -0,0 +1,44 @@
+package com.linkedin.metadata.search.elasticsearch.query.request;
+
+import com.google.common.collect.ImmutableList;
+import com.linkedin.metadata.config.search.SearchConfiguration;
+import com.linkedin.metadata.models.annotation.SearchableAnnotation;
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+import org.elasticsearch.search.aggregations.AggregationBuilder;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+
+public class AggregationQueryBuilderTest {
+
+  @Test
+  public void testGetAggregationsHasFields() {
```

Comment:
Needs a test for the non-`hasX` case for completeness.

Reply:
Okay. I'd really recommend that we slot a backfilling of tests for a bunch of these classes.